src/main/resources/org/sonar/plugins/checkstyle/rules.xml

```diff
@@ -1240,7 +1257,7 @@
     <name><![CDATA[Newline At End Of File]]></name>
     <configKey><![CDATA[Checker/NewlineAtEndOfFile]]></configKey>
     <param key="lineSeparator" type="s[system,crlf,cr,lf,lf_cr_crlf]">
-        <defaultValue>system</defaultValue>
+        <defaultValue>lf_cr_crlf</defaultValue>
```

Comment:
just mention in Mogration notes that without update of Rule, it will work with `system` value - rule need to be recreated or value is manually changed.

Reply:
where should we add those migration notes?

Reply:
I usually put them at bottom of upgrade issue description or as comment.
examples:
https://github.com/checkstyle/sonar-checkstyle/issues/189
https://github.com/checkstyle/sonar-checkstyle/issues/124

Reply:
added comment: https://github.com/checkstyle/sonar-checkstyle/issues/222#issuecomment-502663551